src/app/_utils/setDescriptionParagraph.tsx

```diff
@@ -0,0 +1,14 @@
+type DescriptionProp = string | string[]
+
+export default function setDescriptionParagraph(description: DescriptionProp) {
+  if (Array.isArray(description)) {
+    return (
+      <>
+        {description.map((paragraph, index) => (
+          <p key={index}>{paragraph}</p>
+        ))}
+      </>
+    )
+  }
+  return <p>{description}</p>
+}
```

Comment:
I find it slightly odd to have this bit under `utils` - This feels more like a component if anything.